pyserver/monitor_client.py

```diff
@@ -1,23 +1,25 @@
-
-from flask import Flask, render_template, request, jsonify, Response
+# Copyright (c) 2019 Open Networking Foundation
```

Comment:
please use short header
#
# Copyright 2019-present Open Networking Foundation
#
# SPDX-License-Identifier: Apache-2.0
#

Reply:
Got it

Reply:
Done.